README.md

```diff
@@ -427,7 +427,7 @@ make e2e-test OPERATOR_NAMESPACE=<namespace> E2E_TEST_FLAGS="--deletion-policy=n
 Example commands to run test suite for the dashboard `component` only, with the operator running out of the cluster.
 
 ```shell
-make run-nowebhook
+make run-notebhook
```

Comment:
I think this was correct.

Reply:
i removed this part totally, which has nothing to do with e2e test example.

Reply:
The description is about running the tests suite `with the operator running out of the cluster`, so it seems to make sense in the context, unless we rework the wording.

Reply:
@zdtsw I think it was there to show that using a default `make run` the e2e test most probably won't work as it needs webhook to NOT be deployed. Shouldn't we bring it back?

Reply:
we are having mismtach info on main and rhoai branch
https://github.com/opendatahub-io/opendatahub-operator/blob/rhoai/README.md
does not have the example of "run e2e for dashboard" 
this could be caused by old sync + backport.


for the "make run-nowebhook" probably should be  under section "### Run functional Tests"

i can leave the old content , and get another PR for the "docs updates"